mode/examples/Basics/Math/Map/Map.pyde

```diff
@@ -0,0 +1,26 @@
+'''
+/**
+ * Map.
```

Comment:
This is great; thank you. May I trouble you to please remove the /** java style comment delimiters */ ?